samples/interactive-tutorials/src/test/java/events/PurgeUserEventTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package events;
+
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import util.StreamGobbler;
+
+public class PurgeUserEventTest {
+
+  private String output;
+
+  @Before
+  public void setUp() throws IOException, InterruptedException, ExecutionException {
+    Process exec =
+        Runtime.getRuntime().exec("mvn compile exec:java -Dexec.mainClass=events.PurgeUserEvent");
```

Comment:
To check the response, we should get output from the console and assert then.
This approach is working.

Reply:
I think it's better to avoid exec and just capture the stdout directly. Please see this example for the standard way we do it: https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/storage/cloud-client/src/test/java/com/example/storage/QuickstartSampleIT.java#L47-L67

Reply:
Typically, main is just a stub to make it easier for the user to provide input. That input should be virtually identical to args needed to run the method directly. In general we try to avoid using environment variables to configure snippets, because it's often easier for users to just edit the sample directly and run in whichever way is most comfortable for them (especially for java, it's not always from a terminal).

Reply:
This comment is still open.